### PR TITLE
Fixing JIRA IDENTITY-2619

### PR DESCRIPTION
**Pull request by kasungayan**
Adding the ability of Logout in Tenancy mode, if the Entity Id is used to specify a tenant.

Removing 71f44fcd028bc2cf5b8b6becf86b855cd4bc226f commit.

### CODE REVIEW

File: components/sso-saml/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/processors/LogoutRequestProcessor.java [hunk 1]
```diff
@@ -103,16 +103,25 @@ public SAMLSSOReqValidationResponseDTO process(LogoutRequest logoutRequest, Stri
 
                 issuer = logoutRequest.getIssuer().getValue();
 
-                String[] splitIssuer = issuer.split("@");
-                if (splitIssuer.length == 2 && !splitIssuer[0].trim().isEmpty() && !splitIssuer[1].trim().isEmpty()) {
-                    issuer = splitIssuer[0];
+                if (issuer.contains("@")) {
+                    String[] splitIssuer = issuer.split("@");
```
Comment (pulasthi7) [inline components/sso-saml/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/processors/LogoutRequestProcessor.java]: Check whether the issuer name can have '@' within itself. If so need to split to 2 at the last '@'

Comment (kasungayan) [inline components/sso-saml/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/processors/LogoutRequestProcessor.java]: Fixed with #1281

File: components/sso-saml/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/processors/LogoutRequestProcessor.java [hunk 2]
```diff
@@ -103,16 +103,25 @@ public SAMLSSOReqValidationResponseDTO process(LogoutRequest logoutRequest, Stri
...
+                    if (StringUtils.isNotEmpty(splitIssuer[0]) && StringUtils.isNotEmpty(splitIssuer[1])) {
```
Comment (dinusha-dilrukshi) [inline components/sso-saml/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/processors/LogoutRequestProcessor.java]: Better assign splitIssuer[0] value into a variable and reuse.

Comment (kasungayan) [inline components/sso-saml/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/processors/LogoutRequestProcessor.java]: Fixed with #1281 & 4cbb486f9d093fd4fe779886b14a6063ae3fadf0

**Pull request merged**